apps/ui/src/views/Proposal/Votes.vue

```diff
@@ -113,233 +121,255 @@ watch([sortBy, choiceFilter], () => {
 </script>
 
 <template>
-  <div
-    class="bg-skin-bg sticky top-[112px] lg:top-[113px] z-40 border-b flex space-x-3 font-medium"
-  >
-    <div class="ml-4 max-w-[218px] w-[218px] truncate">Voter</div>
-    <div class="grow w-[40%]">
-      <template v-if="offchainNetworks.includes(proposal.network)"
-        >Choice</template
-      >
-      <UiSelectDropdown
-        v-else
-        v-model="choiceFilter"
-        class="font-normal"
-        title="Choice"
-        gap="12px"
-        placement="left"
-        :items="[
-          { key: 'any', label: 'Any' },
-          { key: 'for', label: 'For', indicator: 'bg-skin-success' },
-          { key: 'against', label: 'Against', indicator: 'bg-skin-danger' },
-          { key: 'abstain', label: 'Abstain', indicator: 'bg-skin-text' }
-        ]"
-      >
-        <template #button>
-          <button
-            class="flex items-center hover:text-skin-link space-x-2"
-            type="button"
-          >
-            <span class="truncate">Choice</span>
-            <IH-adjustments-vertical class="shrink-0" />
-          </button>
-        </template>
-      </UiSelectDropdown>
-    </div>
-    <button
-      type="button"
-      class="hidden lg:flex max-w-[144px] w-[144px] items-center hover:text-skin-link space-x-1 truncate"
-      @click="handleSortChange('created')"
-    >
-      <span>Date</span>
-      <IH-arrow-sm-down v-if="sortBy === 'created-desc'" class="shrink-0" />
-      <IH-arrow-sm-up v-else-if="sortBy === 'created-asc'" class="shrink-0" />
-    </button>
-    <button
-      type="button"
-      class="max-w-[144px] w-[144px] flex items-center justify-end hover:text-skin-link space-x-1 truncate"
-      @click="handleSortChange('vp')"
-    >
-      <span class="truncate">Voting power</span>
-      <IH-arrow-sm-down v-if="sortBy === 'vp-desc'" class="shrink-0" />
-      <IH-arrow-sm-up v-else-if="sortBy === 'vp-asc'" class="shrink-0" />
-    </button>
-    <div class="min-w-[44px] lg:w-[60px]" />
-  </div>
-
-  <UiLoading v-if="!loaded" class="px-4 py-3 block" />
-  <template v-else>
+  <div ref="el" class="overflow-x-auto no-scrollbar">
     <div
```

Comment:
Is this div and the one above can not be merged together ?

Reply:
The one in Proposal.vue?

Reply:
The top one is allowing us to overflow and the child is setting fixed width to be scrolled horizontally, and settings fixed height for `sticky` to work, and a `overflow-x-auto` is preventing this div going off when we scroll down

so merging seems not possible with the conditions we have 🙈

Reply:
There is a solution, added a fix